composer.json

```diff
@@ -21,7 +21,7 @@
   ],
   "require": {
     "php": ">=5.6",
-    "league/oauth2-client": "1.2.0"
+    "league/oauth2-client": "^2.0"
```

Comment:
If you could please change this to `"league/oauth2-client": "~1.0|~2.0"` it would allow me to keep this from bumping up a major version.